src/tensor_api.cpp

```diff
@@ -199,6 +199,33 @@ extern "C" miopenStatus_t miopenSetTensorDescriptor(miopenTensorDescriptor_t ten
     });
 }
 
+extern "C" miopenStatus_t miopenSetTensorDescriptorEx(miopenTensorDescriptor_t tensorDesc,
```

Comment:
`Ex` usually indicates that the function is an extended version of another function. Here we have a function that accepts a different data type. Perhaps it should be called differently. Probably `miopenSetHugeTensorDescriptor` or something.

Reply:
@averinevg I followed the practice that already exists in miopen.h: found "_V2", "V2" and "Ex" functions and chose the latter due to aesthetic reasons. To me, `miopenSetTensorDescriptor64` looks better, but _Ordnung muß seηn_.

Reply:
Fixed in 6dd37fb8f